app/models/loan_request.rb

```diff
@@ -0,0 +1,16 @@
+class LoanRequest < ActiveRecord::Base
+  validates :title, :description, :amount,
+            :requested_by_date, :repayment_begin_date,
+            :repayment_rate, presence: true
+  has_attached_file :picture,
+    :styles => { :large => "960x960",
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.